src/lib/calculation.ts

```diff
@@ -202,94 +217,101 @@ export function calculate(input: TaxFormUserInput): TaxForm {
       return this.r090
     },
     get r106() {
-      return this.r034.reduce((previousSum, currentChild) => {
-        let currentSum = 0
-        const rateJanuaryToMarch = 22.17
-        const age = rodnecislo(currentChild.rodneCislo).age() // TODO edge cases
-        const rateAprilToDecember = age > 6 ? 22.17 : 44.34
+      return round2decimal(
+        this.r034.reduce((previousSum, currentChild) => {
+          let currentSum = 0
+          const rateJanuaryToMarch = 22.17
+          const age = rodnecislo(currentChild.rodneCislo).age() // TODO edge cases
+          const rateAprilToDecember = age > 6 ? 22.17 : 44.34
 
-        if (currentChild.m00 || currentChild.m01) {
-          currentSum += rateJanuaryToMarch
-        }
-        if (currentChild.m00 || currentChild.m02) {
-          currentSum += rateJanuaryToMarch
-        }
-        if (currentChild.m00 || currentChild.m03) {
-          currentSum += rateJanuaryToMarch
-        }
-        if (currentChild.m00 || currentChild.m04) {
-          currentSum += rateAprilToDecember
-        }
-        if (currentChild.m00 || currentChild.m05) {
-          currentSum += rateAprilToDecember
-        }
-        if (currentChild.m00 || currentChild.m06) {
-          currentSum += rateAprilToDecember
-        }
-        if (currentChild.m00 || currentChild.m07) {
-          currentSum += rateAprilToDecember
-        }
-        if (currentChild.m00 || currentChild.m08) {
-          currentSum += rateAprilToDecember
-        }
-        if (currentChild.m00 || currentChild.m09) {
-          currentSum += rateAprilToDecember
-        }
-        if (currentChild.m00 || currentChild.m10) {
-          currentSum += rateAprilToDecember
-        }
-        if (currentChild.m00 || currentChild.m11) {
-          currentSum += rateAprilToDecember
-        }
-        if (currentChild.m00 || currentChild.m12) {
-          currentSum += rateAprilToDecember
-        }
+          if (currentChild.m00 || currentChild.m01) {
+            currentSum += rateJanuaryToMarch
```

Comment:
Netreba zaokruhlit aj tu? Ja viem ze `+` operacia vyzera safe, ale ja som mal problem pri `-` :)

Reply:
Zaokruhlil som toto pri returne

Reply:
`currentSum` by tu mal byt vzdy len integer ak sa nemylim, takze by to malo byt safe

Reply:
Tak pridal som aj `return round2decimal(previousSum + currentSum)` to snad bude stacit :D